code/controllers/subsystem/garbage.dm

```diff
@@ -139,6 +139,13 @@ SUBSYSTEM_DEF(garbage)
 			pass_counts[i] = 0
 			fail_counts[i] = 0
 
+#ifdef EXPERIMENT_515_QDEL_HARD_REFERENCE
+// 1 from the hard reference in the queue, and 1 from the variable used before this
+#define IS_DELETED(datum, _) (refcount(##datum) == 2)
```

Comment:
I wish this had better comments as to why we're doing it

Reply:
What do you have in mind?

Reply:
I'm unsure, maybe just a blurb about how this should be faster then using \ref constantly. it'd be easier to say if ssgarbage was more thoroughly explained somewhere

Reply:
I'll do it later sure but for the record this macro should be removed when `EXPERIMENT_515_QDEL_HARD_REFERENCE` is removed